content/en/network_monitoring/devices/netflow.md

```diff
@@ -83,15 +85,126 @@ From the **Configuration** tab in NetFlow, click **Add Enrichment** to upload th
+| Destination Subdivision Name | The name of the subdivision (such as state or province) associated with the destination IP. |
+| Destination Timezone | The timezone associated with the destination IP. |
+
+### NetFlow Source IP facets
+
+| Facet Name | Facet Description |
+|---|---|
+| Source AS Domain | The domain associated with the Autonomous System (AS) to which the source IP belongs. |
+| Source AS Name | The name of the Autonomous System (AS) to which the source IP belongs. |
+| Source AS Number | The number assigned to the Autonomous System (AS) to which the source IP belongs. |
+| Source AS Route | The route information associated with the Autonomous System (AS) to which the source IP belongs. |
+| Source AS Type | The type of Autonomous System (AS) to which the source IP belongs (such as transit, customer, peer). |
+| Source Application Name | The name of the application associated with the source IP. |
+| Source City Name | The name of the city associated with the source IP. |
+| Source Cloud Provider Name | The name of the cloud provider associated with the source IP. |
+| Source Cloud Provider Region | The region of the cloud provider associated with the source IP. |
+| Source Cloud Provider Service | The service provided by the cloud provider associated with the source IP. |
+| Source Continent Code | The code representing the continent associated with the source IP. |
+| Source Continent Name | The name of the continent associated with the source IP. |
+| Source Country ISO Code | The ISO code representing the country associated with the source IP. |
+| Source Country Name | The name of the country associated with the source IP. |
+| Source IP | The source IP address. |
+| Source Latitude | The latitude coordinate associated with the source IP. |
+| Source Longitude | The longitude coordinate associated with the source IP. |
+| Source MAC | The Media Access Control (MAC) address associated with the source IP. |
+| Source Mask | The subnet mask associated with the source IP. |
+| Source Port | The source port number. |
+| Source Subdivision ISO Code | The ISO code representing the subdivision (such as state or province) associated with the source IP. |
+| Source Subdivision Name | The name of the subdivision (such as state or province) associated with the source IP. |
+| Source Timezone | The timezone associated with the source IP. |
+
+By monitoring these key fields in NetFlow events, organizations can gain visibility into their network infrastructure, optimize performance, and improve security posture.
+
+{{< img src="monitors/monitor_types/netflow/monitor.png" alt="Create a dashboard with NetFlow data" width="100%" >}}
+
+This data is also available in dashboards, notebooks, and more for more precise queries and for correlating with other sources of data. When creating a dashboard with Netflow data, select **NetFlow** as the source in the **Graph your data** section.
```

Comment:
Can we reword this to avoid the awkward `more for more` construction? (also, don't forget to capitalize the F in NetFlow)

Reply:
```suggestion
This data is also available in dashboards and notebooks, enabling precise queries and correlation with other data sources. When creating a dashboard with NetFlow data, select **NetFlow** as the source in the **Graph your data** section.
```